pages/instructorFileEditor/instructorFileEditor.ejs

```diff
@@ -66,14 +78,24 @@
             <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
 
             <div class="card mb-4">
-                <div class="card-header">
+                <div class="card-header bg-primary">
                     <div class="row align-items-center justify-content-between">
                         <div class="col-auto">
-                            <span class="text-monospace font-weight-bolder"><%= fileEdit.fileNameForDisplay %></span>
+                            <span class="text-monospace text-white"><%= fileEdit.fileNameForDisplay %></span>
                         </div>
                         <div class="col-auto collapse <% if ((!fileEdit.failedPush) && (!fileEdit.alertChoice)) { %>show<% } %>" id="buttons-<%= fileEdit.uuid %>">
-                            <a class="btn btn-secondary" data-toggle="collapse" href="#help-<%= fileEdit.uuid %>" role="button" id="help-<%= fileEdit.uuid %>-button">Show help</a>
-                            <button id="file-editor-<%= fileEdit.uuid %>-save-button" name="__action" value="save_and_sync" class="btn btn-primary" disabled>Save and sync</button>
+                            <a id="help-<%= fileEdit.uuid %>-button"
+                               class="btn btn-light btn-sm"
+                               data-toggle="collapse" href="#help-<%= fileEdit.uuid %>"
+                               role="button" aria-labelledby="help-<%= fileEdit.uuid %>-button-label"
+                               aria-expanded="false">
+                                <i class="far fa-question-circle"></i>
+                                <span id="help-<%= fileEdit.uuid %>-button-label">Show help</span>
```

Comment:
```suggestion
                               role="button" aria-expanded="false">
                                <i class="far fa-question-circle"></i>
                                Show help
```
Actually we probably don't want `aria-labelledby` here. In general we should only use aria to encode info that's not already conveyed by native html semantics, and here the link already has a native "label" (its own link text).

Reply:
Thx for the tip, testing it with NVDA, the label gets detected with or without `aria-labelledby`